webapp_frontend/src/components/pages/Registration.tsx

```diff
@@ -103,12 +98,12 @@ export default function Registration(): ReactElement {
         setPasswordConfirmation(value);
     }
 
-    const reviewPasswordMatch = ():void => {
+    const reviewPasswordMatch = (): void => {
         setPasswordsMatch(password === passwordConfirmation);
     }
 
     return (
-        <Container>
+        <Container className="h-100" style={{position: "relative"}}>
```

Comment:
Should be a class instead of inline style. Maybe class name pos-rel?

Reply:
As this PR is still marked as a draft this one is still in progress and not finished. Was just experimenting and will be replaced.

Reply:
just saved it in a WIP commit because I had no time left to work on it